lib/child-writer/writer-proxy.js

```diff
@@ -157,6 +158,13 @@ return isElevated().then((elevated) => {
       message: 'User did not grant permission.'
     }, () => {
       process.exit(EXIT_CODES.CANCELLED);
+    }).catch({
+      message: 'No polkit authentication agent found.'
```

Comment:
Fair enough. Would it be worth chucking some code-comments in here to reflect that fact?
(i.e. that the `sudo-prompt` error messages need to be checked if we ever upgrade it)